pkg/performanceprofile/controller/performanceprofile/hypershift/hypershift.go

```diff
@@ -90,6 +90,14 @@ func (ci *ControlPlaneClientImpl) Update(ctx context.Context, obj client.Object,
 	return ci.Client.Update(ctx, obj, opts...)
 }
 
+func (ci *ControlPlaneClientImpl) Delete(ctx context.Context, obj client.Object, opts ...client.DeleteOption) error {
```

Comment:
please add in the commit message a line or few about the intended usage for the `Delete` API

Reply:
I'm pretty sure I did - from the commit message:
```
The `Delete` call deletes the `ConfigMap` that holds an encoded data of the object that was given for deletion.
```

Reply:
you indeed described what the `Delete` call would do, but is not obvious which flow needs `Delete` (and how we managed so far?)

Reply:
anyway, not really blocking